test/TestConfig/scripts/testKitGen/testKitGen.pl

```diff
@@ -79,9 +82,9 @@
 			. "                            If the modesXml is not provided, the program will try to find modes.xml under projectRootDir/TestConfig/resources.\n"
 			. "  --ottawaCsv=<path>        Path to ottawa.csv file.\n"
 			. "                            If the ottawaCsv is not provided, the program will try to find ottawa.csv under projectRootDir/TestConfig/resources.\n"
-			. "  --buildList=<paths>       Comma separated project paths to search playlist.xml. The paths are relative to projectRootDir.\n"
```

Comment:
Why `buildList` is removed? Is it intended?

Reply:
updated